storage/imx6ull-flash/flashdev.c

```diff
@@ -30,6 +30,9 @@
 #define ECCN_BITFLIP_STRENGHT 14  /* ECC strength for data chunk */
 #define ECCN_DATA_SIZE        512 /* Data chunk size */
 
+#define CHUNK_IS_META(chunkidx)                ((chunkidx) == 0)
+#define CHUNK_IS_UNCORECTABLE(chunkidx, flips) ((CHUNK_IS_META(chunkidx) && (flips) > ECC0_BITFLIP_STRENGHT) || \
```

Comment:
UNCORECTABLE -> UNCORRECTABLE

Also - missing `!` before CHUNK_IS_META in second line.

IMHO, this should be written as an inline function (much more readable, less likely to make an error):
```c
static inline int chunk_is_uncorrectable(chunkidx, flips)
{
  if (CHUNK_IS_META(chunkidx)) {
    return flips > ECC0_BITFLIP_STRENGHT;
  }
  else {
     return flips > ECCN_BITFLIP_STRENGHT;
  }
}
```

If You really want it to be a function-like macro, maybe use ternary operator, eg.:
```c
#define CHUNK_IS_UNCORECTABLE(chunkidx, flips) (CHUNK_IS_META(chunkidx) ? ((flips) > ECC0_BITFLIP_STRENGHT)) : ((flips) > ECCN_BITFLIP_STRENGHT)
```

Reply:
Okay, I can make it an inline function.